cmake/develop_dependencies.cmake

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+#######################################################################
+# Declare project dependencies
+#######################################################################
+
+FetchContent_Declare( ENDFtk
+    GIT_REPOSITORY  https://github.com/njoy/ENDFtk
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
+FetchContent_Declare( dimwits
+    GIT_REPOSITORY  https://github.com/njoy/DimensionalAnalysis
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
+FetchContent_Declare( lipservice
+    GIT_REPOSITORY  https://github.com/njoy/lipservice
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
+FetchContent_Declare( njoy_c_bindings
+    GIT_REPOSITORY  https://github.com/njoy/njoy_c_bindings
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
+FetchContent_Declare( tclap-adapter
+    GIT_REPOSITORY  https://github.com/njoy/tclap-adapter
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
+FetchContent_Declare( utility
+    GIT_REPOSITORY  https://github.com/njoy/utility
+    GIT_TAG         origin/master
+    GIT_SHALLOW     TRUE
+    )
+
```

Comment:
Will need to add `FetchContent_Declare( leapr...` here.

Reply:
Thank you! That was my mistake to forget that. I updated the cmake files to have LEAPR information in them, but I'm getting a slight problem when I try to compile NJOY21, that it complains about LEAPR's dependence on ENDFtk. 

Are you able to compile this branch at all? At first I thought it was due to a discrepancy between NJOY21's GIT_TAG of ENDFtk and LEAPR's GIT_TAG for ENDFtk but resolving that didn't immediately seem to make a difference

Reply:
I'm not able to compile this. After some investigation, I also can't compile the master branch of LEAPR, I think that is the problem. Are you able to compile LEAPR?